client/blocks/upload-image/docs/example.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { partial } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import UploadImage from '../';
+import { AspectRatios } from 'state/ui/editor/image-editor/constants';
+
+export default class UploadImageExample extends Component {
+	state = {
+		isUploading: false,
+		uploadedImageDataUrl: null
+	};
+
+	render() {
+		const { isUploading, uploadedImageDataUrl } = this.state;
+
+		return (
+			<div className="docs__design-assets-group">
+				<h3>Default Upload Image</h3>
+				<UploadImage
+					isUploading={ isUploading }
+					onImageEditorDone={
```

Comment:
it's pretty simple to add a method to the class instead of creating the arrow function straight as a property.

```es6
editImage = imageBlob => this.setState( {
  uploadedImageDataUrl: URL.createObjectURL( imageBlob ),
  isUploading: true,
} );

render() {
  return (
    <UploadImage
      isUploading={ isUploading }
      onImageEditorDone={ this.editImage }
    />
  );
}
```